patches/minecraft/net/minecraft/world/gen/feature/ConfiguredFeature.java.patch

```diff
@@ -0,0 +1,11 @@
+--- a/net/minecraft/world/gen/feature/ConfiguredFeature.java
++++ b/net/minecraft/world/gen/feature/ConfiguredFeature.java
+@@ -15,7 +15,7 @@
+ import org.apache.logging.log4j.LogManager;
+ import org.apache.logging.log4j.Logger;
+ 
+-public class ConfiguredFeature<FC extends IFeatureConfig, F extends Feature<FC>> implements IDecoratable<ConfiguredFeature<?, ?>> {
++public class ConfiguredFeature<FC extends IFeatureConfig, F extends Feature<FC>> extends net.minecraftforge.registries.ForgeRegistryEntry<ConfiguredFeature<?, ?>>  implements IDecoratable<ConfiguredFeature<?, ?>> {
```

Comment:
There is an extra space being added before implements

Reply:
Fix